lib/index.js

```diff
@@ -58,8 +58,10 @@ api.db = null;
 // shared collections cache
 api.collections = {};
 
+// Note: api.writeOptions is deprecated
+// and will be removed in release 9.0
 // default database write options
-api.writeOptions = bedrock.config.mongodb.writeOptions;
+api.writeOptions = {...bedrock.config.mongodb.writeOptions};
```

Comment:
I think we're going to have to not make this shallow copy -- it will change the API.

```suggestion
api.writeOptions = bedrock.config.mongodb.writeOptions;
```